easybuild/easyconfigs/v/VBZ-Compression/VBZ-Compression-1.0.1-gompi-2020b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'VBZ-Compression'
+version = '1.0.1'
+
+homepage = 'https://github.com/nanoporetech/vbz_compression'
+description = "VBZ compression HDF5 plugin for nanopolish"
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+
+source_urls = [
+    'https://github.com/nanoporetech/vbz_compression/archive',
+    'https://github.com/lemire/streamvbyte/archive',
+]
+sources = [
+    {'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ},
+    {'download_filename': 'v0.4.1.tar.gz', 'filename': 'streamvbyte-v0.4.1.tar.gz'},
+]
+checksums = [
+    '3c140fb4f60f8451db68768457c59b10aadba4c33f1b1a1eadb4439345b68192',  # VBZ-Compression-1.0.1.tar.gz
+    '4c4e53134a60b0b06816d3faa7dcde28c3e5e8a656dd415d16d80ae6e3d39fcc',  # streamvbyte-v0.4.1.tar.gz
+]
+
+builddependencies = [('CMake', '3.18.4')]
+
+dependencies = [
+    ('HDF5', '1.10.7'),
+    ('zstd', '1.4.5'),
+]
+
+preconfigopts = "rmdir %(builddir)s/vbz_compression*/third_party/streamvbyte && "
+preconfigopts += "mv %(builddir)s/streamvbyte-* %(builddir)s/streamvbyte && "
+preconfigopts += "mv %(builddir)s/streamvbyte %(builddir)s/vbz_compression*/third_party/. && "
```

Comment:
Why the two-step move for `streamvbyte`? Why not do:

```
mv %(builddir)s/streamvbyte-* %(builddir)s/vbz_compression*/third_party/streamvbyte
```

Maybe I'm missing something?

Reply:
see:
https://github.com/easybuilders/easybuild-easyconfigs/pull/13536#discussion_r680867615
and 
https://github.com/easybuilders/easybuild-easyconfigs/pull/13536#discussion_r680893756

Reply:
See https://github.com/easybuilders/easybuild-easyconfigs/pull/13536#discussion_r680893756

Reply:
Ah, I overlooked the `*` in the destination, interesting that this doesn't work. Fine, not a big problem, of course.